src/com/mediaengine/rpc/rpc_cdm_mediaengine_handler.cc

```diff
@@ -19,6 +19,23 @@
 #include <cdm_logging.h>
 #include <opencdm_xdr.h>
 
+#ifdef OCDM_SDP_PROTOTYPE
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/socket.h>
+#include <sys/ioctl.h>
+#include <sys/mman.h>
+#include "linux/ion.h"
```

Comment:
Where is this header being included from?

Reply:
ion.h is included from the intermediate file: tmp/sysroots/imx8mqevk/usr/include/linux/ion.h. This is copied from the kernel source: ./drivers/staging/android/uapi/ion.h. 

Is that ok?

Note that, since this is a system include, I will replace the "" by <> in the next pull request.